src/styles/snake/antisnake.scss

```
@@ -0,0 +1,42 @@
+.antisnake {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+
+  &__cell {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 100px;
+    color: #fff;
+    height: 300px;

```

You should create a variable for сell size.
```suggestion
    height: $cell-size;

```